app/pages/landingIndustryUser.js

```diff
@@ -0,0 +1,220 @@
+import React, { Component } from 'react';
+import { Button, Accordion, Card, Container, Col, Row } from 'react-bootstrap';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import { graphql } from 'react-relay';
+import Organisations from '../containers/Organisations/Organisations';
+
+const organizations = [
+  { orgName: 'Rio Tinto Alcan', facilities: ['Facility1', 'Facility2'] },
+  {
+    orgName: 'Shell Coporation',
+    facilities: ['Facility1', 'Facility2', 'Facility3']
+  },
+  { orgName: 'Shanghai Eletric', facilities: ['Facility1', 'Facility2'] }
+];
+
+class landingIndustryUser extends Component {
+  state = {
+    organizations: [],
+    facilities: []
+  };
+  static query = graphql`
+  query landingIndustryUserQuery {
+    query {
+     ...Organisations_query
+    }
+  }
+  `;
+  renderAccordian = () => {
+    const organizationArr = [];
+    let facilitiesArr = [];
+    for (let i = 0; i < organizations.length; i++) {
+      for (let j = 0; j < organizations[i].facilities.length; j++) {
+        facilitiesArr.push(
```

Comment:
use the [`Array.prototype.map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function instead of iterating over an array and pushing in another one